addon/components/frost-list.js

```diff
@@ -114,6 +136,36 @@ export default Component.extend({
     })
   },
 
+  @readOnly
+  @computed('componentKeyNamesForTypes')
+  isAnyTypedItemExpansion (componentKeyNamesForTypes) {
```

Comment:
Can we rename this to isAnyItemExpansion please?

Reply:
I can rename this to `isAnyItemExpansion` and change the logic to return true if typed item expansion or default item expansion, but then we wouldn't be able to use it for the `35px` padding in front of the list item because that logic is only show the padding if there is expansion present in the list but not for this particular list item.